tree/30_generic_methods/file_key_value_present_option.cf

```diff
@@ -39,7 +39,7 @@ bundle agent file_key_value_present_option(file, key, value, separator, option)
       "args"              slist => { "${file}", "${key}", "${value}","${separator}", "${option}" };
 
       "report_param"      string => join("_", args);
-      "class_prefix"      string => canonify("${old_class_prefix}_${report_param}");
+      "class_prefix"      string => canonify("file_key_value_present_${report_param}");
```

Comment:
changing this signature breaks the 
"inner_class_prefix" string => canonify("file_key_value_present_${file}_${report_param}"); 
in file_ensure_key_value_option.cf